ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2862,6 +2834,15 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-static-ip
+- as: vsphere-ipi-static-ip-tp-f60-longduration-cloud
+  cron: 3 2 15 2,4,6,8,10,12 *
+  steps:
+    cluster_profile: vsphere-connected-2
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    test:
+    - chain: openshift-e2e-test-clusterinfra-qe-longrun
+    workflow: cucushift-installer-rehearse-vsphere-ipi-static-ip
```

Comment:
I think the CPMS cases will failed as this bug  https://issues.redhat.com/browse/OCPBUGS-44750, should we choose another workflow?